packages/editor/src/pages/Auth/index.tsx

```diff
@@ -141,19 +137,32 @@ class AuthPage extends React.Component<IProps, IState> {
         };
       }
 
-      if (this.state.publicKey && !this.state.isIE) {
-        const random = generateCryptoSafeRandom();
-
-        sessionStorage.setItem(
-          AUTH_PAGE_SESSION_STORAGE_KEYS.auth_state,
-          random.toString(),
-        );
+      if (this.state.publicKeyString && !this.state.isIE) {
```

Comment:
are you using isIE to decide if we're in the sad frame or the authing frame?

Reply:
See comment above